app/com/linkedin/drelephant/spark/fetchers/SparkRestClient.scala

```diff
@@ -143,6 +144,7 @@ object SparkRestClient {
     }
 
     val objectMapper = new ObjectMapper() with ScalaObjectMapper
+    objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
```

Comment:
Thanks for catching that @shkhrgpt! I'll remove the conflicting line when I do a final rebase and cleanup.